xml/templates/message_templates/payment_or_refund_notification_html.tpl

```diff
@@ -160,7 +160,7 @@
             </td>
           </tr>
     {/if}
-    {if $contributeMode eq'direct' and !$isAmountzero}
+    {if $credit_card_number}
```

Comment:
we would never have a card number here if it's not been used for a payment so we don't need contributeMode or to check the amount

Reply:
Agree